<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ion Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="MatplotlibWidget" name="mplwidget_up"/>
       <widget class="MatplotlibWidget" name="mplwidget_down"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_File"/>
    <addaction name="actionLoad_Folder"/>
    <addaction name="actionClean_Load_File"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionColorMapSettingDialog"/>
   </widget>
   <widget class="QMenu" name="menuCopy">
    <property name="title">
     <string>Copy</string>
    </property>
    <addaction name="actionCopy_Image"/>
    <addaction name="actionCopy_Slice"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
   <addaction name="menuCopy"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionLoad_File">
   <property name="text">
    <string>Load File</string>
   </property>
  </action>
  <action name="actionLoad_Folder">
   <property name="text">
    <string>Load Folder</string>
   </property>
  </action>
  <action name="actionClean_Load_File">
   <property name="text">
    <string>Clean Load File</string>
   </property>
  </action>
  <action name="actionRange">
   <property name="text">
    <string>Range</string>
   </property>
  </action>
  <action name="actionColorMapSettingDialog">
   <property name="text">
    <string>Color Map</string>
   </property>
  </action>
  <action name="actionNormalize_by_DV">
   <property name="text">
    <string>Normalize by DV</string>
   </property>
  </action>
  <action name="actionCopy_Image">
   <property name="text">
    <string>Copy Image</string>
   </property>
  </action>
  <action name="actionCopy_Slice">
   <property name="text">
    <string>Copy Slice</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header>matplotlibwidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
